lib/util/AuthApp/actions.js

```diff
@@ -67,13 +66,50 @@ export default class AuthAppActions extends AuthAppGetters {
     }
 
     async setPassword(password) {
-        await (await this.getTitleHeading()).waitForDisplayed({ timeout: 3000});
+        await (await this.getTitleHeading()).waitForDisplayed({ timeout: 3000 });
         await expect(await this.getTitleHeading()).toHaveText(PasswordHeading);
         await (await this.gePasswordInput()).waitForClickable({ timeout: 30000 });
         await (await this.gePasswordInput()).click();
         await (await this.gePasswordInput()).addValue(password);
         await (await this.getSubmitButton()).click();
     }
 
+    /*
+    ** Reset account
+    */
 
+    async cannotAccessAccount() {
+        await (await this.getTitleHeading()).waitForDisplayed({ timeout: 3000 });
+        await expect(await this.getTitleHeading()).toHaveText(SignInHeading);
+
```

Comment:
Maybe this empty lines should be removed. Or if you want to adopt this style, all functions should respect this